android-exoplayer/src/main/java/com/brentvatne/exoplayer/CustomAdaptiveTrackSelection.java

```diff
@@ -0,0 +1,39 @@
+package com.brentvatne.exoplayer;
+
+import com.google.android.exoplayer2.Format;
+import com.google.android.exoplayer2.source.TrackGroup;
+import com.google.android.exoplayer2.trackselection.AdaptiveTrackSelection;
+import com.google.android.exoplayer2.upstream.BandwidthMeter;
+import com.google.common.collect.ImmutableList;
+
+public class CustomAdaptiveTrackSelection extends AdaptiveTrackSelection {
+
+        public CustomAdaptiveTrackSelection(TrackGroup group, int[] tracks, BandwidthMeter bandwidthMeter) {
+                super(group, tracks, bandwidthMeter);
+        }
+
+        @Override
+        protected boolean canSelectFormat(Format format, int trackBitrate, float playbackSpeed, long effectiveBitrate) {
+
+                long _effectiveBitrate;
+
+                if(ReactExoplayerView.qualityCounter <= 3 && ReactExoplayerView.isNotTrailer) {
```

Comment:
let make this 2 instead of 3

Reply:
and no need to use `isNotTrailer ` , already have `youboraParams` 

if no youboraParams then its trailer